application/composer.json

```diff
@@ -32,13 +32,12 @@
         "proj4js/proj4js": "^2.4",
         "mapbender/mapbender": "^3.3.4-RC1",
 
-        "mapbender/coordinates-utility": "^1.1",
         "mapbender/digitizer": "^1.4",
         "select2/select2": "^4.0",
         "components/font-awesome": "^4.0",
         "wheregroup/cookieconsent": "^3",
         "wheregroup/sassc-binaries": "^0.0.1",
-        "components/bootstrap": "^3"
+        "twbs/bootstrap": "^5.0"
```

Comment:
IMO, the bootstrap dependency should only live in the mapbender core repository, not in the starter, since it is widely used in the core as well and removing it in custom projects will not work. 

In fact, the old dependency still exists in the core repo and is yet untouched: https://github.com/mapbender/mapbender/blob/feature/bootstrap-update/composer.json#L41